### PR TITLE
Allow Fluentd to be optional when installing Verrazzano

### PR DESCRIPTION
**Pull request by bobdonat**
# Description

This pull request adds a Fluentd enabled flag to the verrazzano custom resource for it to be optionally installed.  The default is to install Fluentd.

Fixes VZ-3088

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [x] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: platform-operator/helm_config/charts/verrazzano/values.yaml [hunk 1]
```diff
@@ -68,6 +68,9 @@ logging:
   name: fluentd
   # NOTE: The fluentd-kubernetes-daemonset image now comes from the bill of materials file (verrazzano-bom.json).
 
+fluentd:
+  enabled: true
```
Comment (michael-cico) [inline platform-operator/helm_config/charts/verrazzano/values.yaml:72]: From what I read in the PR description, should this be `false` for the managed cluster defaults?  If so you'd need to update the `values.managed-cluster.yaml` file I think also.

Comment (bobdonat) [reply platform-operator/helm_config/charts/verrazzano/values.yaml:72]: per @shihchang Fluentd is always installed for the managed cluster.  I did a managed cluster install and Fluentd was installed.

Comment (bobdonat) [reply platform-operator/helm_config/charts/verrazzano/values.yaml:72]: Sorry PR description was wrong

Comment (shihchang) [reply platform-operator/helm_config/charts/verrazzano/values.yaml:72]: The default value should be the same, true, on admin and managed; as long as the managed has containers running, the fluentd would have logs to collect.

File: platform-operator/apis/verrazzano/v1alpha1/verrazzano_types.go [hunk 2]
```diff
@@ -346,6 +346,8 @@ type RancherComponent struct {
 
 // FluentdComponent specifies the Fluentd DaemonSet configuration
 type FluentdComponent struct {
+	// +optional
```
Comment (shihchang) [inline platform-operator/apis/verrazzano/v1alpha1/verrazzano_types.go:349]: Should the Enabled have some minimum godoc if different components have different default values?

File: platform-operator/helm_config/charts/verrazzano/values.yaml (hunk 1 above)
Comment (bobdonat) [reply platform-operator/helm_config/charts/verrazzano/values.yaml:72]: default is true in all cases

File: platform-operator/apis/verrazzano/v1alpha1/verrazzano_types.go (hunk 2 above)
Comment (bobdonat) [reply platform-operator/apis/verrazzano/v1alpha1/verrazzano_types.go:349]: the default is true for all components

**Review by shihchang — DISMISSED** [review]

**Review by michael-cico — APPROVED** [review]

**Pull request merged**